README.md

```diff
@@ -2,6 +2,94 @@
 
 The complete code of conduct can be read at [code-of-conduct.socialtables.com](http://code-of-conduct.socialtables.com)
```

Comment:
We should remove this as it will no longer be available there.

Also, should we remove the CNAME file?